certbot/main.py

```diff
@@ -405,6 +405,17 @@ def _init_le_client(config, authenticator, installer):
 
     return client.Client(config, acc, authenticator, installer, acme=acme)
 
+def _want_delete(config):
+    """Does the user want to delete their now-revoked certs?
+
+    """
+    prompt = (
+            'Would you like certbot to delete the revoked certificate '
+            'and related files on disk? Warning: your severs could be '
```

Comment:
How could your servers be left misconfigured? Aren't they also left misconfigured if the cert is revoked? Is this because it's worse to not be able to find a file than it is to find a broken cert file?